app/controllers/users/passwords_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Users
+  class PasswordsController < Devise::PasswordsController
+    def create
+      if User.find_by(email: resource_params['email'], provider: 'oktaoauth')
+        redirect_to "#{ENV['OKTA_URL']}/enduser/settings"
```

Comment:
please add this env variable as rails config
+ and use URI.join for forming url